onadata/apps/api/viewsets/xform_viewset.py

```diff
@@ -749,6 +761,40 @@ def destroy(self, request, *args, **kwargs):
 
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    @action(methods=['GET', 'POST'], detail=True)
+    def version(self, request, *args, **kwargs):
+        xform = self.get_object()
+        version_id = kwargs.get('version_id')
+        job_uuid = kwargs.get('job_uuid')
+        version = None
```

Comment:
Are we passing the `job_uuid` as a query? something like the following
`/api/v1/forms/<form-id>/version/<version>?job_uuid=<job_uuid>`

Reply:
Yes, I've updated this piece of code to retrieve the value from the `query_params`